samples/maps/demo/flowmap-flight-routes/demo.js

```diff
@@ -0,0 +1,197 @@
+(async () => {
+    const mapData = await fetch(
+        'https://code.highcharts.com/mapdata/countries/gb/gb-all.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.mapChart('container', {
+
+        title: {
+            text: 'Highmaps basic flight routes demo'
+        },
+
+        subtitle: {
+            text: 'Demo of Highcharts map with flowmap'
+        },
+
+        mapNavigation: {
+            enabled: true
+        },
+
+        tooltip: {
+            formatter: function () {
+                return this.point.id + (
+                    this.point.lat ?
+                        '<br>Lat: ' + this.point.lat +
+                        ' Lon: ' + this.point.lon : ''
+                );
+            }
+        },
+
+        series: [{
+            // Use the gb-all map with no data as a basemap
+            mapData,
+            name: 'Great Britain',
+            borderColor: '#a8a8a8',
+            nullColor: 'rgba(200, 200, 200, 0.3)',
+            showInLegend: false,
+            states: {
+                inactive: {
+                    enabled: false
+                }
+            }
+        }, {
+            // Specify cities using lat/lon
+            type: 'mappoint',
+            id: 'cities',
+            name: 'Cities',
+            dataLabels: {
+                format: '{point.id}'
+            },
+            // Use id instead of name to allow for referencing points later using the flowmap series.
+            data: [{
+                id: 'London',
+                lat: 51.507222,
+                lon: -0.1275
+            }, {
+                id: 'Birmingham',
+                lat: 52.483056,
+                lon: -1.893611
+            }, {
+                id: 'Leeds',
+                lat: 53.799722,
+                lon: -1.549167
+            }, {
+                id: 'Glasgow',
+                lat: 55.858,
+                lon: -4.259
+            }, {
+                id: 'Sheffield',
+                lat: 53.383611,
+                lon: -1.466944
+            }, {
+                id: 'Liverpool',
+                lat: 53.4,
+                lon: -3
+            }, {
+                id: 'Bristol',
+                lat: 51.45,
+                lon: -2.583333
+            }, {
+                id: 'Belfast',
+                lat: 54.597,
+                lon: -5.93
+            }, {
+                id: 'Lerwick',
+                lat: 60.155,
+                lon: -1.145,
+                dataLabels: {
+                    align: 'left',
+                    x: 5,
+                    verticalAlign: 'middle'
+                }
+            }]
+        }, {
+            type: 'flowmap',
+            linkedTo: 'cities',
+            minWeight: 8,
+            maxWeight: 12,
```

Comment:
That's quite heavy compared to the usual flight routes map found on the Internet™️. The usual is 1px or 2px lines. Besides, we're missing a demo of how to display flows as simple lines - the three proposed demos all have fat arrows with `growTowards`.